Documentation/getting-started-guide.md

```diff
@@ -22,6 +22,41 @@ func main() {
 }
 ```
 
+### Use systemd notification mechanism
```

Comment:
I agree and suggest to move this to https://github.com/coreos/rkt/blob/master/Documentation/using-rkt-with-systemd.md.